scripts/compile-sierra-contract-class.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -e
+
+print_usage() {
+cat <<EOF 
+Expected 3 arguments, received $1.
```

Comment:
This prints only the first argument. To print the number of arguments you should use:

```suggestion
Expected 3 arguments, received $#.
```

Reply:
The number of arguments is passed as an argument to the function. So actually `$1`, refers to that argument. If you prefer, I can use `$#` directly.

Reply:
Actually, `$1` refers to the number of arguments (it's passed as an argument to the function). I can use `$#` directly if you prefer

Reply:
Oh sorry, you are right. You can dismiss this comment.